### ISSUE TITLE
BlueHost :: Builder Sync :: I can't delete the only section in BB

### ISSUE THREAD

**Issue by FaycalBoutam** [elegantthemes/Divi-Beta#942]
### Problem:

There's always a section added to the builder and you can't delete it. It seems I can do that now:

![91](https://cloud.githubusercontent.com/assets/16414613/22736406/dcfbdc60-edf5-11e6-9855-c824d0d66e1f.gif)

**Comment by fikrirasyid** [comment]
@FaycalBoutam can you recheck this again on latest `575-fix-heartbeat-content-smashing-wip`? I'm able to replicate this issue last night. This morning I'm pulling the latest changes and I'm no longer able to replicate the issue. 

If you're still able to replicate the issue, can you mention step by step to replicate it? Thanks

**Comment by timayhelen** [comment]
@FaycalBoutam The latest version has been uploaded to the test site, so you can just test it there and let us know if it still happens.

**Comment by FaycalBoutam** [comment]
@fikrirasyid @timayhelen Here's how you can reproduce it:
1 Create new page.
2 Left click the VB button.
3 Right click the Edit Page link on the admin bar to open the second tab with the BB.
4 Load any layout and start deleting all section including the last one on the VB tab (you can't delete it of course).
5 Go to the BB tab, and now you'll be able to delete it.

![92](https://cloud.githubusercontent.com/assets/16414613/22781906/8240c570-eebc-11e6-97d6-7be5007f3a20.gif)

**Comment by fikrirasyid** [comment]
@FaycalBoutam have you cleared your browser cache before trying it? If you haven't cleared your cache prior to test it, it'd be really helpful if you clearing your cache then test it again. I've tried following your steps multiple times, and the issue doesn't appear. I've verified it by asking @lots0logs to replicate the issue and he couldn't replicate it as well. These are the screencast of my attempt of replicating the issue on the test site: https://www.dropbox.com/s/cep92yszpdf8dkm/divi-beta-942-issue.mov?dl=0
